README.md

```diff
@@ -105,7 +105,7 @@ class Editor extends React.Component {
     const { editorState } = this.state;
     if (!CodeUtils.hasSelectionInBlock(editorState)) return 'not-handled';
 
-    this.onChange(CodeUtils.onTab(evt, editorState));
+    this.onChange(CodeUtils.onTab(evt, editorState, 2));
```

Comment:
This probably shouldn't be hard coded? Rather than being provided as an argument it should ideally be detected—that's what the `getIndentation.js` utils is for afaict.

Can we somehow infer this instead of hardcoding it? Why do we need this argument here?

Reply:
Hmmm. it's not hardcoded. You can set it with props if you want, or you mean we should show something like `CodeUtils.onTab(evt, editorState, indentDepth)` here?

And again. I don't how `getIndentation` can help here. The case I see for it is to detect current line indentation and set next line similar indentation on that data, like `insertNewLine` does